app/assets/javascripts/browsered/form-validation.js

```diff
@@ -4,20 +4,20 @@ const chargeValidation = require('../../../utils/charge-validation')
 const { toggleSubmitButtons } = require('./helpers')
 const { submitWithWorldpay3dsFlexDdcResult } = require('./worldpay-3ds-flex-ddc')
 
-var init = function () {
-  var form = document.getElementById('card-details')
-  var formInputs = Array.prototype.slice.call(form.querySelectorAll('input'))
-  var countrySelect = document.getElementById('address-country')
-  var postcodeInput = document.getElementById('address-postcode')
-  var cardInput = document.getElementById('card-no')
-  var corporateCardMessageElement = document.getElementById('corporate-card-surcharge-message')
-  var paymentSummaryAmountValue = document.getElementById('amount').getAttribute('data-amount')
-  var paymentSummaryBreakdownElement = document.getElementById('payment-summary-breakdown')
-  var paymentSummaryBreakdownAmountElement = document.getElementById('payment-summary-breakdown-amount')
-  var paymentSummaryCorporateCardFeeElement = document.getElementById('payment-summary-corporate-card-fee')
-  var paymentSummaryAmountElement = document.getElementById('amount')
-  var errorSummary = document.getElementsByClassName('govuk-error-summary')[0]
-  var logger = { info: function () { } }// replace with console to see output
+const init = function () {
+  const form = document.getElementById('card-details')
```

Comment:
`const` won't work on IE browser < 11 https://gds-way.cloudapps.digital/manuals/programming-languages/js.html#when-to-use-standardx. We should check if we are still getting traffic for this version and possibly block IE browser < 11 too.

Reply:
Is the IE issue just for the `getElementById` calls? Or should I just revert all the `const` changes in this file?

Reply:
OK - I think I've reverted all the relevant ones. I think we can keep the changes in `utils`, `start.js` and the tests?